drivers/can/can_loopback.c

```diff
@@ -81,6 +81,12 @@ static void tx_thread(void *arg1, void *arg2, void *arg3)
 			continue;
 		}
 
+#ifndef CONFIG_CAN_ACCEPT_RTR
```

Comment:
There is a mixed use of #ifndef and `if (!IS_ENABLED(CONFIG_CAN_ACCEPT_RTR)) {` in the PR. Just wondering if at least some of the ifndef and ifdef's could be replaced by `IS_ENABLED`.

Reply:
I know. I have deliberately used `IS_ENABLED()` where I found it to make the code more readable, but stuck with `#ifdef` where I believe the opposite was true.